<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCalibrateParameter</class>
 <widget class="QWidget" name="QCalibrateParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCalibrateParameter</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="S1abs">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>106</y>
     <width>175</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="parameter" stdset="0">
    <string>bks</string>
   </property>
  </widget>
  <widget class="QLabel" name="K">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>106</y>
     <width>175</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="parameter" stdset="0">
    <string>bks</string>
   </property>
  </widget>
  <widget class="QLabel" name="A">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>202</y>
     <width>175</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="parameter" stdset="0">
    <string>bks</string>
   </property>
  </widget>
  <widget class="QLabel" name="B">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>202</y>
     <width>175</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="parameter" stdset="0">
    <string>bks</string>
   </property>
  </widget>
  <widget class="QLabel" name="C">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>298</y>
     <width>175</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="parameter" stdset="0">
    <string>bks</string>
   </property>
  </widget>
  <widget class="QLabel" name="L">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>298</y>
     <width>175</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="parameter" stdset="0">
    <string>bks</string>
   </property>
  </widget>
  <widget class="QLabel" name="H">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>394</y>
     <width>175</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="parameter" stdset="0">
    <string>bks</string>
   </property>
  </widget>
  <widget class="QLabel" name="I">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>394</y>
     <width>175</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="parameter" stdset="0">
    <string>bks</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>491</y>
     <width>421</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Paramter_com">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="Calibrate_method">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>10</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_time">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_lot">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>70</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_expired_2">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>100</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_usedTime">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>130</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_method_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>校准方法：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_time_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>校准时间：</string>
     </property>
     <property name="tests" stdset="0">
      <string>t</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_lot_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>校准品批号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_expired_2_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>校准品有效期：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_usedTime_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>校准参数有效时间：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Parameter_current">
    <widget class="QLabel" name="Assay_status">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>100</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Assay_pos">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>40</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Assay_name">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>10</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Assay_register_time">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>130</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Assay_type">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>70</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Parameter_source">
     <property name="geometry">
      <rect>
       <x>304</x>
       <y>100</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_type">
     <property name="geometry">
      <rect>
       <x>304</x>
       <y>40</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="reagent_lot">
     <property name="geometry">
      <rect>
       <x>304</x>
       <y>10</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_expired">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Assay_name_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>项目名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Assay_pos_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>位置：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Assay_type_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>试剂类型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Assay_status_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>试剂状态：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Assay_register_time_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>初次登记时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="reagent_lot_label">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>10</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>试剂批号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_type_label">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>40</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>校准类型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Calibrate_expired_label">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>有效期：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Assay_register_time_label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Parameter_source_label">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>100</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>系数由来：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="S1abs_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>71</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>S1Abs</string>
   </property>
  </widget>
  <widget class="QLabel" name="K_label">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>71</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QLabel" name="A_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>167</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="B_label">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>167</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QLabel" name="C_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>263</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_label">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>263</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QLabel" name="H_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>359</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="I_label">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>359</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
